<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fenetre</class>
 <widget class="QMainWindow" name="Fenetre">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>149</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fenetre</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*Reste*/
QWidget {
	background-color: rgb(55,58,71);
	color: #FFF;
}

/*QLineEdit*/
QLineEdit {
	border: 2px solid rgb(37,39,48);
	border-radius: 7px;
	color: #FFF;
	padding-left: 20px;
	padding-right: 20px;
	background-color: rgb(34,36,44);
}

QLineEdit:hover {
	border: 2px solid rgb(48,50,62);
}

QLineEdit:focus {
	border: 2px solid rgb(85,170,255);
	background-color: rgb(43,45,56);
}


</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_connexion">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/*QPushButton*/
QPushButton {
	color: #FFF;
	border: 2px solid rgb(0,0,0);
	background-color: rgb(81,84,103);
	border-radius: 7px
}

QPushButton:hover {
	border: 2px solid rgb(255,255,255);
	background-color: rgb(53,55,67);
	border-radius: 7px
}</string>
    </property>
    <property name="text">
     <string>Connexion</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_pswd">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Master Password :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pswd_verif">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Verification :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_show">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: none;
	border: none;
	background-image: url(&quot;C:/Users/Jonathan/Desktop/Gestionnaire_MotDePasse - Copie/Images/Logo_Show_password-inv.png&quot;)l;

}

QPushButton:hover {
	background-image: url(&quot;C:/Users/Jonathan/Desktop/Gestionnaire_MotDePasse - Copie/Images/Logo_Show_password-inv_b_w.png&quot;);
}

/* # est utilisé pour donéne l'objet*/

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>401</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_pswd_verif">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_pswd">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>401</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Serif</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font-weight: bold
}</string>
    </property>
    <property name="text">
     <string>Login base des mots de passes</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>edit_pswd</tabstop>
  <tabstop>edit_pswd_verif</tabstop>
  <tabstop>btn_show</tabstop>
  <tabstop>btn_connexion</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
